src/Aspire.Hosting.Python/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: Experimental("ASPIREHOSTINGPYTHON001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
```

Comment:
Just to make sure, is the intention really to mark _**ALL**_ APIs as experimental as opposed to just the new ones? Presumably we would want to keep the other ones not as experimental and instead just obsolete them?

Reply:
cc: @eerhardt @DamianEdwards @sebastienros

Reply:
From what I am seeing in the PublicAPI files, all previous have been obsoleted, so everything is new. And everything is 
- one resource (PythonAppResource)
- two extension methods (AddPythonApp)

Reply:
Yes, I think we just make ALL APIs in this library Experimental for 9.0. All the previous APIs are obsolete. And all the new APIs are experimental.